pkg/profilecreator/mcp.go

```diff
@@ -12,6 +13,56 @@ import (
 	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 )
 
+// GetMCPSelector returns a label that is unique to the target pool, error otherwise
+func GetMCPSelector(pool *mcfgv1.MachineConfigPool, clusterPools []*mcfgv1.MachineConfigPool) (map[string]string, error) {
```

Comment:
I wonder if this function should just return `key, value, error` since the expected size of the `map[string]string` it returns now is always 1

Reply:
I decided to keep it that way since the receiver of this function result is the PerformanceProfile which expects the same type: map[string]string for the MCP selector. I take it is some kind of "pattern".